src/pages/court-cases/[courtCaseId]/index.tsx

```diff
@@ -106,7 +117,8 @@ export const getServerSideProps = withMultipleServerSideProps(
       }
     }
 
-    const courtCase = await getCourtCaseByOrganisationUnit(dataSource, +courtCaseId, currentUser)
+    // Fetch the record from the database after updates
+    courtCase = await getCourtCaseByOrganisationUnit(dataSource, +courtCaseId, currentUser)
```

Comment:
Post-update fetch the court case from the database to ensure that we use the updated court case value to render the UI.